core/tests/behavior/append.rs

```diff
@@ -71,14 +71,16 @@ pub async fn test_append_create_append(op: Operator) -> Result<()> {
 }
 
 /// Test append to a directory path must fail.
-pub async fn test_append_with_dir_path(op: Operator) -> Result<()> {
+pub async fn test_append_to_dir_path(op: Operator) -> Result<()> {
     let path = format!("{}/", uuid::Uuid::new_v4());
     let (content, _) = gen_bytes();
 
     let res = op.append(&path, content).await;
     assert!(res.is_err());
     assert_eq!(res.unwrap_err().kind(), ErrorKind::IsADirectory);
 
+    op.delete(&path).await.expect("delete file must success");
```

Comment:
I prefer to implement `setup` and `teardown` for our tests instead of add delete for every test.

Reply:
Of course, it's better to do it this way.

Reply:
> Of course, it's better to do it this way.

Would you like to give it a try? If not, I will reject this PR.

Reply:
I'll give it a try. It's ok to close this.